app/views/index.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Chitter!</h1>
+
+<% if !@peeps.empty? %>
```

Comment:
I was expecting to see something which would let a user create a peep. As it stands they can just sign up.